App.js

```diff
@@ -42,12 +45,12 @@ const config = {
 };
 
 
-const addChatRouteIfNecessary = (state) => {
+const fixState = (state) => {
     // we don't want to add Chat route for small screens
     if (checkIsSmallScreen(Dimensions.get('window').width)) {
       return 
     }
-    if (!_.find(state.routes, r => r.name === 'Chat')) {
+    if (!find(state.routes, r => r.name === 'Chat')) {
```

Comment:
I know this is just POC, but I would keep the underscore here, we use that in the App